packages/braintree-integration/src/braintree-accelerated-checkout/braintree-accelerated-checkout-customer-strategy.ts

```diff
@@ -105,7 +105,9 @@ export default class BraintreeAcceleratedCheckoutCustomerStrategy implements Cus
         const { authenticationState, profileData } = await triggerAuthenticationFlow(customerId);
 
         if (authenticationState === BraintreeConnectAuthenticationState.SUCCEEDED) {
-            const addresses = profileData.addresses.map(this.mapPayPalConnectToBcAddress);
+            const addresses = profileData.addresses.map((address) =>
```

Comment:
```suggestion
            const addresses = profileData.addresses.map(address =>
```

Reply:
There was the issue with ESLint but I will double check. Thanks!

Reply:
<img width="1468" alt="Screenshot 2023-08-03 at 14 33 33" src="https://github.com/bigcommerce/checkout-sdk-js/assets/25133454/1294cfd3-829e-4855-b216-d63b406c0a4a">